orb-qr-link/src/user_data.rs

```diff
@@ -16,6 +16,9 @@ pub struct UserData {
     /// Personal Custody Package version.
     #[serde(default = "pcp_version_default")]
     pub pcp_version: u16,
+    /// Whether the app should perform a app-centric signup.
+    #[serde(default = "default_false")]
```

Comment:
This can be simplified, because the default value for `bool` is `false`. `default_false` function can be removed.

```suggestion
    #[serde(default)]
```